precompiles/avs/types.go

```diff
@@ -228,7 +228,7 @@ func (p Precompile) GetTaskParamsFromInputs(_ sdk.Context, args []interface{}) (
 	if !ok || (callerAddress == common.Address{}) {
```

Comment:
Given the CallerAddress is sdk.AccAddress,  so the type assertion  to common.Address is not reasonable. plase check all such code and the return error : 

> callerAddress, ok := args[0].(common.Address)